release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -4181,4 +4181,30 @@ $ oc adm release info 4.12.37 --pullspecs
 [id="ocp-4-12-37-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-12-39"]
+=== RHSA-2023:5677 - {product-title} 4.12.39 bug fix and security update
+
+Issued: 2023-10-18
+
+{product-title} release 4.12.39, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2023:5677[RHSA-2023:5677] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2023:5679[RHSA-2023:5679] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.12.39 --pullspecs
+----
+
+[id="ocp-4-12-39-bug-fixes"]
+==== Bug fixes
+
+* Previously, CoreDNS would crash if a user created an EndpointSlice port without a port number. With this update, validation was added to CoreDNS so that it will no longer crash in this situation.  (link:https://issues.redhat.com/browse/OCPBUGS-20144[*OCPBUGS-20144*])
```

Comment:
Looks like you have an extra space here
```suggestion
* Previously, CoreDNS would crash if a user created an EndpointSlice port without a port number. With this update, validation was added to CoreDNS so that it will no longer crash in this situation. (link:https://issues.redhat.com/browse/OCPBUGS-20144[*OCPBUGS-20144*])
```

Reply:
Also while I am here some suggestions:
```suggestion
* Previously, CoreDNS would crash if an EndpointSlice port was created without a port number. With this update, validation was added to CoreDNS so it will no longer crash in this situation.  (link:https://issues.redhat.com/browse/OCPBUGS-20144[*OCPBUGS-20144*])
```